### DISCUSSION TITLE
Github Action for spell check on a pull request

### DISCUSSION THREAD

**Discussion by uditgaurav** [community/community#25384] [Actions]
<p>Hello all,<br>
I just want some action that can do a spell check on a pull request when asked for like If I comment on a pull request with something like <code>/spell-check</code> then it should do a spell-check over the content of pull request. The content may contain some .md files or comments over code and may ignore codes.<br>
I get to know the first part of this and I can trigger a workflow on pull request comment<br>
basis using this:</p>
<pre><code class="lang-auto">name: Litmus-CI
on:
  issue_comment:
    types: [created, edited]
jobs:
  tests:
    if: contains(github.event.comment.html_url, '/pull/') &amp;&amp; contains(github.event.comment.body, '/spell-check')
    runs-on: ubuntu-latest
    steps:
</code></pre>
<p>looking for the other part if we have such actions or something similar?<br>
Please share if you know some actions around this.<br>
Thanks!</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/uditgaurav">@uditgaurav</a>,</p>
<p>There is no official actions to do a spell-check over pull request content.<br>
According to the description <a href="https://github.com/microsoft/terminal/pull/4799" rel="nofollow noopener">here</a>:</p>
<p>Spell checking can occur:</p>
<ul>
<li>on commits (pushing to a branch)</li>
<li>on PRs (creating a pull request)</li>
</ul>
<p>You can try to use action <a href="https://github.com/check-spelling/check-spelling" rel="nofollow noopener">check-spelling/check-spelling</a></p>
<p>Thanks.</p>
